includes/fbproductfeed.php

```diff
@@ -777,7 +777,7 @@ private function create_feed() {
 			return $feed_id;
 		}
 
-		private function create_upload( $facebook_feed_id ) {
+		public function create_upload( $facebook_feed_id ) {
```

Comment:
I suspect these methods have not been used - i.e. this PR is reactivating dormant code. Just noting that so we can test/derisk appropriately.

Reply:
these are used by `WC_Facebook_Product_Feed::sync_all_products_using_feed`, but I suspect the same as I didn't find a path where it was being executed